lib/src/code_field/actions/ignore.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/widgets.dart';
+
+class IgnoreIntent extends Intent {
```

Comment:
Can we use this one? https://api.flutter.dev/flutter/widgets/DoNothingIntent-class.html